dev-tools/globals.ts

```diff
@@ -22,25 +23,27 @@ let TeamValidatorAsync = require('../team-validator-async');
 
 type GenderName = 'M' | 'F' | 'N' | '';
 type StatName = 'hp' | 'atk' | 'def' | 'spa' | 'spd' | 'spe';
-type StatsTable = {hp: number, atk: number, def: number, spa: number, spd: number, spe: number};
-type SparseStatsTable = {hp?: number, atk?: number, def?: number, spa?: number, spd?: number, spe?: number};
-type SparseBoostsTable = {hp?: number, atk?: number, def?: number, spa?: number, spd?: number, spe?: number, accuracy?: number, evasion?: number};
+type StatsTable = {[key:string]: number, hp: number, atk: number, def: number, spa: number, spd: number, spe: number};
+type SparseStatsTable = Partial<StatsTable>;
+type BoostsTable = {[key:string]: number, atk: number, def: number, spa: number, spd: number, spe: number, accuracy: number, evasion: number};
```

Comment:
Ditto these two. Just keep the ts-ignores.

Reply:
Is there really not some better way of doing this, though? We're talking _at least_ 100 ignores...

Reply:
You can cast to `{[key:string]: number}` if you want, in the `for` part of the loop?

Reply:
Alternatively: Maybe we should just have a `STAT_NAMES` array somewhere, and then just `for (const stat of STAT_NAMES) {` instead of `for..in`?